src/main/java/io/strimzi/kafka/bridge/mqtt/kafka/BridgeKafkaProducer.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.bridge.mqtt.kafka;
+
+import io.strimzi.kafka.bridge.mqtt.config.KafkaConfig;
+import org.apache.kafka.clients.producer.*;
+import org.apache.kafka.common.serialization.StringSerializer;
+import java.util.Properties;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+
+/**
+ * Represents a Kafka producer for the Bridge.
+ */
+public class BridgeKafkaProducer<K, V> {
+
+
+    private final int ackLevel;
```

Comment:
Would it be better if we used an enum instead of int here?

We could even add some comments in the enum explaining the different ACK levels

Let me know what you think!

Reply:
yes, that would be better

Reply:
I think this should be a `String` in order to be prepared for using `all` other than `0` and `1` as possible values.

Reply:
"all" is equivalent to -1.

Reply:
let's keep the semantic we have in Kafka, 0, 1, all ... not moving to something different. You never know if at some point Kafka will start using -1 with a specific meaning.

Reply:
Anyway, I have changed this.
Now I am using enum as suggested by @kyguy .